fvm/evm/evm_test.go

```diff
@@ -151,6 +152,7 @@ func TestEVMRun(t *testing.T) {
 				require.Equal(t, blockEventPayload.Hash, txEventPayload.BlockHash)
 				require.Equal(t, blockEventPayload.Height, txEventPayload.BlockHeight)
 				require.Equal(t, blockEventPayload.TotalGasUsed, txEventPayload.GasConsumed)
+				require.Greater(t, blockEventPayload.TotalGasUsed, uint64(0))
```

Comment:
I believe this should equal to the `txEventPayload.GasConsumed`

Reply:
This assertion is already on the line above:
```go
require.Equal(t, blockEventPayload.TotalGasUsed, txEventPayload.GasConsumed)
```
I simply added one more assertion to be safe, rather than sorry :sweat_smile: 
Note that for `EVM.run` this was actually already working.